python/ray/test/test_utils.py

```diff
@@ -118,6 +118,7 @@ def _pid_alive(pid):
     """
     try:
         os.kill(pid, 0)
+        return True
```

Comment:
The lack of this return statement was a bug meaning that any calls to `wait_for_pid_to_exit` always succeeded.